src/Ledger/Transaction.lagda

```diff
@@ -91,22 +89,23 @@ This function must produce a unique id for each unique transaction body.
 \end{code}
 \emph{Transaction types}
 \AgdaTarget{TxBody, txins, txouts, txfee, txvldt, txwdrls, txup, txADhash, txsize, txid, TxWitnesses, vkSigs, scripts, Tx, body, wits, txAD}
+\begin{AgdaSuppressSpace}
 \begin{code}
   record TxBody : Set where
     field txins      : ℙ TxIn
           txouts     : Ix ↛ TxOut
-          txcerts    : List DCert
-          mint       : Value
           txfee      : Coin
+          mint       : Value
           txvldt     : Maybe Slot × Maybe Slot
+          txcerts    : List DCert
           txwdrls    : Wdrl
+          txvote     : List GovVote
+          txprop     : List GovProposal
           txup       : Maybe Update
```

Comment:
Not sure of the logic of the (re)ordering of these fields.  Would it make sense to pick a standard, e.g., lexicographic, ordering? ...or would that result in ungrouping things that should be grouped?

Reply:
I wanted to order them in order of importance from my biased viewpoint and keeping related fields next to each other, with the fields that related to the serialization at the bottom. We can bike-shed that when we have some more time.